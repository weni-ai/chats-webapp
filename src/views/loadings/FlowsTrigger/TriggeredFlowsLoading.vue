<template>
  <section class="rooms-table-loading">
    <header class="rooms-table-loading__categories">
      <UnnnicSkeletonLoading />
      <UnnnicSkeletonLoading />
      <UnnnicSkeletonLoading />
      <UnnnicSkeletonLoading />
      <UnnnicSkeletonLoading />
    </header>
    <section
      class="rooms-table-loading__row"
      v-for="row in 4"
      :key="`row-${row}`"
    >
      <UnnnicSkeletonLoading />
      <UnnnicSkeletonLoading />
      <UnnnicSkeletonLoading />
      <UnnnicSkeletonLoading />
      <UnnnicSkeletonLoading />
    </section>
  </section>
</template>

<script>
export default {
  name: 'RoomsTableFiltersLoading',
};
</script>

<style lang="scss" scoped>
.rooms-table-loading {
  display: flex;
  flex-direction: column;

  overflow: hidden auto;

  &__categories,
  &__row {
    display: grid;
    grid-template-columns: repeat(5, 1fr);
    gap: $unnnic-spacing-sm;

    padding: $unnnic-spacing-ant $unnnic-spacing-sm;

    & > * {
      display: flex;
      height: $unnnic-font-size-body-gt + $unnnic-line-height-md;
    }
  }

  &__categories {
    margin-bottom: $unnnic-spacing-nano;
  }
}
</style>
