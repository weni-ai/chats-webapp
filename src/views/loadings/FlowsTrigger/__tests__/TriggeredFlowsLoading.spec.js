import { mount } from '@vue/test-utils';
import TriggeredFlowsLoading from '../TriggeredFlowsLoading.vue';
import { beforeEach, describe, expect } from 'vitest';

const createWrapper = () => {
  return mount(TriggeredFlowsLoading);
};

describe('TriggeredFlowsLoading.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = createWrapper();
  });

  it('to match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
