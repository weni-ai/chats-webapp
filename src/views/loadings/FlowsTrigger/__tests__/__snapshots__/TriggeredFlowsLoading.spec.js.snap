// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TriggeredFlowsLoading.vue > to match snapshot 1`] = `
"<section data-v-a20b9b94="" class="rooms-table-loading">
  <header data-v-a20b9b94="" class="rooms-table-loading__categories"><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></header>
  <section data-v-a20b9b94="" class="rooms-table-loading__row"><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></section>
  <section data-v-a20b9b94="" class="rooms-table-loading__row"><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></section>
  <section data-v-a20b9b94="" class="rooms-table-loading__row"><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></section>
  <section data-v-a20b9b94="" class="rooms-table-loading__row"><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-a20b9b94=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></section>
</section>"
`;
